evaluator/evaluator.py

```diff
@@ -270,31 +270,40 @@ def evaluate_vmaas(self, system_platform, cur, loop=None):
         new_cves = set()
         mitigated_cves_ids = set()
         unmitigated_cves_ids = set()
+        inc_cves_ids = set()
+        dec_cves_ids = set()
```

Comment:
variables for CVE ids we will be really updating in caches as when a CVE has rule associated it doesn't matter if VMaaS reports that is un/mitigated as in case of unmitigation we'd be reporting it for 2nd time to the cache and for case of mitigation .. it's really not mitigated as there's still the rule reporting we're vulnerable